src/usr/local/www/guiconfig.inc

```diff
@@ -492,15 +492,19 @@ function gentitle($title) {
 	}
 }
 
-function genhtmltitle($title) {
+function genhtmltitle($title, $links=array()) {
 
 	// If the array contains only one element, there are no breadcrumbs, so don't
 	// add anything else
 	if (count($title) > 1) {
 		$bc = '<ol class="breadcrumb">';
 
-		foreach ($title as $el) {
-			$bc .= '<li>'.$el.'</li>';
+		foreach ($title as $idx => $el) {
+			if (strlen($links[$idx]) > 0) {
+				$bc .= '<li><a href="https://' . $_SERVER['SERVER_ADDR'] . '/' . $links[$idx] . '">' .$el.'</a></li>';
```

Comment:
@jim-p Your are correct - I had been thinking that the remote client would not know what "/services_dhcp.php" is when clicking a link. But there is "magic" somewhere (I guess in the client browser) that fills in the protocol, URL and port bits based on the current page.
I pushed a fix to that.